src/main/java/core/basesyntax/service/operation/OperationHandler.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.service.operation;
+
+import core.basesyntax.dao.OperationDao;
+import core.basesyntax.dao.impl.OperationDaoImpl;
+import core.basesyntax.model.FruitTransaction;
+
+public interface OperationHandler {
+    OperationDao operationDao = new OperationDaoImpl();
```

Comment:
It's not recommended to have a field with an implementation in an interface. Consider using dependency injection to provide the required implementation.